h/resources.py

```diff
@@ -140,6 +140,42 @@ def __getitem__(self, pubid):
         return organization.logo
 
 
+class GroupResource(object):
+    def __init__(self, group, request):
+        self.request = request
+        self.group = group
+        self.links_service = self.request.find_service(name='group_links')
+
+    @property
+    def links(self):
+        return self.links_service.get_all(self.group)
+
+    @property
+    def organization(self):
```

Comment:
I'm hoping that the way this property works it can take advantage of whatever loading is in play with SQLAlchemy for relations. That is, I made it a property (versus defining in the constructor) such that the `OrganizationResource` won't be created until it's asked for, meaning that if the `organization` relation hasn't been loaded by SQLA yet, it will only lazy load when needed here.

On the flip side, this should also work with eagerly-loaded organization relations. I _think_!